src/uppy.js

```diff
@@ -183,6 +190,19 @@ const getDebugLogger = (options) => {
  * @param {object} options
  */
 const getOptionsMiddleware = (options) => {
+  let s3Client = null
+  if (options.providerOptions.s3) {
+    const S3 = require('aws-sdk/clients/s3')
+    const config = options.providerOptions.s3
+    s3Client = new S3({
```

Comment:
any particular reason we are now creating the client here, and not in [s3.js](https://github.com/transloadit/uppy-server/blob/s3%2Fmultipart/src/server/controllers/s3.js)?

Reply:
I made some changes to how s3 client is implemented in #84 which is currently not merged, but would have to be replicated here.